vslib/src/sai_vs_interfacequery.cpp

```diff
@@ -202,6 +206,73 @@ void unittestChannelThreadProc()
     SWSS_LOG_NOTICE("exit VS unittest channel thread");
 }
 
+void processFdbEntriesForAging()
+{
+    MUTEX();
+
+    SWSS_LOG_ENTER();
+
+    uint32_t current = (uint32_t)time(NULL);
+
+    // find aged fdb entries
+
+    for (auto it = g_fdb_info_set.begin(); it != g_fdb_info_set.end();)
+    {
+        sai_attribute_t attr;
+
+        attr.id = SAI_SWITCH_ATTR_FDB_AGING_TIME;
```

Comment:
>SAI_SWITCH_ATTR_FDB_AGING_TIME [](start = 18, length = 30)

since this is switch attribute, why not move it out of the for loop and skip the for loop if aging is not enable?

Reply:
this is obtained from switch id, in case of multiple switches where it can be different, or i can get this from first item on the list

Reply:
ok

Reply:
im leaving this as is